experimental/dds/tree2/.dependency-cruiser-known-violations.json

```diff
@@ -471,5 +471,24 @@
 			"src/feature-libraries/editable-tree-2/index.ts",
 			"src/feature-libraries/editable-tree-2/proxies/index.ts"
 		]
+	},
+	{
+		"type": "cycle",
```

Comment:
Appears to be a duplicate of existing cycle between editable-tree and editable-tree-2